src/vos/vos_aggregate.c

```diff
@@ -383,9 +386,9 @@ vos_agg_sv(daos_handle_t ih, vos_iter_entry_t *entry,
 			 * Highest epoch is uncommitted, skip it and continue
 			 * checking on next lower epoch.
 			 */
-			D_DEBUG(DB_EPC, "Skip uncommitted at epoch:"DF_X64"\n",
-				entry->ie_epoch);
-			break;
+			D_DEBUG(DB_EPC, "Hit uncommitted single value at epoch:"
+				DF_X64"\n", entry->ie_epoch);
+			return -DER_TX_BUSY;
```

Comment:
The comments above need be revised, the original design is to skip the uncommitted epoch and continue on the next lower epoch.

Reply:
Ok, I will fix that.